main/modes/quickSettings/menuQuickSettingsRenderer.c

```diff
@@ -0,0 +1,166 @@
+#include "menuQuickSettingsRenderer.h"
+#include "menu_utils.h"
+
+#include "hdw-tft.h"
+#include "shapes.h"
+#include "fill.h"
+#include "wsg.h"
+#include "font.h"
+#include "menu.h"
+#include "palette.h"
+#include "esp_log.h"
+
+#include <malloc.h>
+#include <stdint.h>
+#include <inttypes.h>
+
+//==============================================================================
+// Defines
+//==============================================================================
+
+/// The width of the panel, centered in the screen
+#define PANEL_W (TFT_WIDTH - 60)
+
+/// The height of the panel, from the top of the screen
+#define PANEL_H 60
+
+/// The panel corner-radius
+#define PANEL_R 15
+
+#define TEXT_MARGIN 10
+
+#define PANEL_BG_COLOR   c333
+#define PANEL_BOX_COLOR  c000
+#define PANEL_TEXT_COLOR c000
+
+#define ICON_W      16
+#define ICON_H      16
+#define ICON_BOX_PX 2
+
+//==============================================================================
+// Structs
+//==============================================================================
+
+typedef struct
+{
+    /// @brief The menu item label these
+    const char* label;
+    /// @brief The icon to draw when the setting is on
+    const wsg_t* onWsg;
+    /// @brief The icon to draw when the setting is off
+    const wsg_t* offWsg;
```

Comment:
If you do like the option of having the full range of settings, you can still use the on/off images, and have the off image only apply to the min setting.

Reply:
This has been done now! And I think it's much better like this